espnet2/asr/frontend/s3prl.py

```diff
@@ -50,14 +50,10 @@ def __init__(
             normalize=frontend_conf.get("normalize", False),
             extra_conf=frontend_conf.get("extra_conf", None),
         )
+        if getattr(upstream.upstream, "model", None):
+            if getattr(upstream.upstream.model, "feature_grad_mult", None):
+                upstream.upstream.model.feature_grad_mult = 1.0
         upstream.eval()
-        if getattr(
-            upstream.upstream, "model", None
-        ) is not None and upstream.upstream.model.__class__.__name__ in [
-            "Wav2Vec2Model",
-            "HubertModel",
-        ]:
-            upstream.upstream.model.encoder.layerdrop = 0.0
```

Comment:
Why are these lines removed?

Reply:
I think this is because S3PRL already sets `encoder_layerdrop=0` when initializing an upstream, so we no longer need to do this in ESPnet. Am I right?

Reply:
Yes, it is correct.